modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1018,8 +1041,18 @@ struct InferROI: public cv::detail::KernelTag {
 
                         IE::Blob::Ptr this_blob = extractBlob(*ctx, 1);
 
-                        req.SetBlob(*(ctx->uu.params.input_names.begin()),
-                                IE::make_shared_blob(this_blob, toIE(this_roi)));
+                        // NB: In case importNetwork preprocessing must be
+                        // passed as SetBlob argument.
+                        if (ctx->uu.params.kind == cv::gapi::ie::detail::ParamDesc::Kind::Load) {
+                            req.SetBlob(*(ctx->uu.params.input_names.begin()),
+                                    IE::make_shared_blob(this_blob, toIE(this_roi)));
```

Comment:
Indent

Reply:
What is wrong with the indent ?

Reply:
I would do like this:
```
req.SetBlob(*(ctx->uu.params.input_names.begin()),
            IE::make_shared_blob(this_blob, toIE(this_roi)));
```

Am I wrong? Is tab better?

Reply:
I just put `enter` after the comma on the first line, I believe it is correct ident.

Reply:
BTW, it isn't relevant anymore

Reply:
Ok.